tsl/src/bgw_policy/continuous_aggregate_api.c

```diff
@@ -29,15 +45,327 @@ policy_continuous_aggregate_get_mat_hypertable_id(const Jsonb *config)
 	return mat_hypertable_id;
 }
 
+static int64
+cagg_get_from_config(const Dimension *dim, const Jsonb *config, const char *json_label,
```

Comment:
Agree with @erimatnor here. The function is specifically for retrieving a timestamp, so the name and parameters should reflect that.

Reply:
modified.